README.md

```diff
@@ -66,6 +66,14 @@ Following are the implementations managed by third-parties adopting the standard
   * Machine API Operator, https://github.com/openshift/machine-api-operator/tree/master
   * Machine-controller-manager, https://github.com/gardener/machine-controller-manager/tree/cluster-api
 
+## Versioning, Maintenance, and Compatibility
+
+- We follow [Semantic Versioning (semver)](https://semver.org/).
```

Comment:
In order to make it clearer what qualifies as a "backwards incompatible change" that would trigger a major version bump, would it makes sense to add some examples here?
* You can't run the new controllers in some existing management clusters.
* You can't manage some existing clusters deployed by the prior version of Cluster API.
* Your automation scripts or yaml for interacting with Cluster API will no longer work.

Reply:
Given that the readme was meant to provide only a short summary, would you like to take as an action item to follow-up with a more in depth versioning document? This is where I took inspiration for these points https://github.com/kubernetes-sigs/controller-runtime/blob/master/VERSIONING.md